Twig/Extension/MediaExtension.php

```diff
@@ -169,7 +169,7 @@ public function thumbnail($media = null, $format, $options = array())
 
         $options = array_merge($defaultOptions, $options);
 
-        $options['src'] = $provider->generatePublicUrl($media, $format);
+        $options = $provider->getHelperProperties($media, $format, $options);
```

Comment:
Maybe we can call both methods and merge the `$options` to be BC with old and newer versions?